invenio_queues/__init__.py

```diff
@@ -6,7 +6,126 @@
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-"""Invenio module for collecting statistics."""
+
+r"""Invenio module for managing queues.
+
+This guide will show you how to get started with Invenio-Queues. It assumes
+that you already have knowledge of Flask applications and Invenio modules.
+
+It will then explain key topics and concepts of this module.
+
+Getting started
+---------------
+
+You will learn how to register queues and to interact with it. To begin with,
+you need to setup your virtual environment and install this module.
+
+You need an application to work with, that can be created with the following
+commands in a Python shell:
+
+>>> from flask import current_app
```

Comment:
```suggestion
.. code-block:: python
    from flask import current_app
```

Skip also the `>>>` in general, I think by default `py.test` will try to run them as doctests (which we don't want to get into).